app/pdfs/passengers_PDF.rb

```diff
@@ -0,0 +1,34 @@
+require 'prawn/table'
+
+class PassengersPDF < Prawn::Document
+  def initialize(passengers, filters)
+    super(page_layout: :landscape, page_size: "A3" )
```

Comment:
1. Does this `page_layout: :landscape` deal actually work? Because I saw the PDF and the page layout seems to be `:letter`. 
2. What does `page_size: "A3" do?

Reply:
1. Yes, it does work.
2. http://www.papersizes.org/a-paper-sizes.htm. But don't use that, 11x17 is `:tabloid`

Reply:
Sorry, that should be `'TABLOID'`